pkg/apply/event/event.go

```diff
@@ -75,92 +81,73 @@ type ErrorEvent struct {
 	Err error
 }
 
-//go:generate stringer -type=ApplyEventType
-type ApplyEventType int
+//go:generate stringer -type=ActionGroupEventType
+type ActionGroupEventType int
 
 const (
-	ApplyEventResourceUpdate ApplyEventType = iota
-	ApplyEventCompleted
+	Started ActionGroupEventType = iota
+	Finished
 )
 
+type ActionGroupEvent struct {
+	GroupName string
+	Action    ResourceAction
+	Type      ActionGroupEventType
+}
+
 //go:generate stringer -type=ApplyEventOperation
 type ApplyEventOperation int
 
 const (
-	ServersideApplied ApplyEventOperation = iota
+	NoneApply ApplyEventOperation = iota
```

Comment:
Maybe change `NonApply` to `ApplyUnspecified`. `NonApply` seems to have the similar meaning as `Unchanged`.